src/character-set.html

```diff
@@ -242,7 +242,7 @@ <h4 id="chars_pseudo-scripts">Pseudo-scripts</h4>
    <p>In almost all mathematical contexts, pseudo-script characters should
    be associated
    with a base expression using explicit script markup in MathML.  For
-   example, the preferred encoding of <span>&#x201c;x prime&#x201d;</span> is</p>
+   example, the preferred encoding of <q>x prime</q> is</p>
```

Comment:
@dginev  (or anyone else)  `character-set.html` only appears here as davidcarlisle/mathml/gh-pages was rebased against w3c so had the change from `<span>`  to `<q>` applied. The intention of the rebase was to make the files mostly the same and minimise diffs. And character-set.html in the gh-pages branches in w3c and davidcarlisle forks are in fact now identical.

It doesn't affect the actual merge, git will no doubt skip the "already applied commit", but I don't understand what the github UI diff is showing here?

Reply:
This is resolved (and the diffs from that commit have gone)  (I use the fetch upstream GUI on my fork at gihub) That said I still don't understand what that diff is showing. commandline tools showed  (as far as I can tell) that the head of the gh-pages branch at w3c, davidcarlisle, and my local clones all had that `<q>` commit applied.